README.md

```diff
@@ -382,6 +382,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [OAuth](https://crystal-lang.org/api/OAuth.html) - OAuth consumer (Crystal stdlib)
  * [OAuth2](https://crystal-lang.org/api/OAuth2.html) - OAuth2 client (Crystal stdlib)
  * [OpenSSL](https://crystal-lang.org/api/OpenSSL.html) - bindings to libssl (Crystal stdlib)
+ * [Socks](www.sockscr.com) - SOCKS proxy
```

Comment:
Can we use gitlab link instead? Github/Gitlab ones are validated for uniqueness by tests.

Reply:
I can. I was going to make a homepage with docs and stuff.